saleor/graphql/core/types/common.py

```diff
@@ -146,6 +147,22 @@ class CheckoutError(Error):
     )
 
 
+class OrderFromCheckoutCreateError(Error):
```

Comment:
I think for new mutation we could move it to the file where the mutation is defined? Since we have a file per mutation that would be pretty clean.